buildpack/telemetry/appdynamics_telegraf_output.py

```diff
@@ -0,0 +1,126 @@
+"""The module is used in runtime to convert metrics from telegraf for appdynamics Machine Agent"""
+import json
+import logging
+import requests
+from requests.exceptions import ConnectionError
+
+
+METRIC_TREE_BASE_NODE = "Custom Metrics|mxapp"
+AGGR_TYPE = "OBSERVATION"
+
+
+def filter_last(payload):
```

Comment:
Add some comments to explain the limitation of the AppDynamics metrics API, that we need to ensure we just pass the last entry.

Reply:
Done